config/locales/exceptions.en.yml

```diff
@@ -2163,7 +2163,13 @@ en:
         <<: *defaults
         code: LH_gateway_timeout
         status: 504
-
+      benefits_education:
+        outside_working_hours:
+          <<: *defaults
+          title: External service unavailable
+          detail: 'Lighthouse GI Bill Status is not available outside of working hours'
+          code: 114
```

Comment:
This is the same code # used when raising an exception from EVSS.  I'm assuming it's a good idea to use the same code number.